### PR TITLE
Welcome Rotom, PO's YouTube data presenter.

### PR DESCRIPTION
**Pull request by NightfallAlicorn**

### CODE REVIEW

File: scripts/youtube.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                        length = response.items[0].contentDetails.duration
+                            .toLowerCase().substr(2).replace("h", "h ").replace("m", "m ").replace("s", "s"),
+                        uploader = response.items[0].snippet.channelTitle,
+                        likes = parseInt(response.items[0].statistics.likeCount, 10),
+                        dislikes = parseInt(response.items[0].statistics.dislikeCount, 10),
+                        ratio = Math.round(likes / (likes + dislikes) * 100);
+                    youtubeBot.sendAll("Title: {0}, Length: {1}, Uploader: {2}, Likes: {3}%".format(title, length, uploader, ratio), channel);
+                });
+            } catch (error) {
+                youtubeBot.sendHtmlAll("YouTube video data load failed.");
```
Comment (PokeWorldBW) [inline scripts/youtube.js]: I'm not sure you need to tell the whole server this (also why html?)

**Comment by Arburator** [comment]
error isn't sent to Indigo?

**Comment by NightfallAlicorn** [comment]
Sorted. I was up late coding this last night. I'm surprised I missed quite a few minor things.

Also, i don't know Indi wants all the error messages there. I've attempted to simulate an error with entering a made up video ID, no error occurred.

**Comment by IceKirby** [comment]
Or rather, is it even necessary to broadcast the error? Could just send to the person who posted the link (and maybe to Indigo too).

**Comment by Arburator** [comment]
also sending error as sendall and not sendmessage(src, message, channel)

**Comment by Fuzzysqurl** [comment]
What point would the error message serve? It's not like we can fix it if it errors.

**Comment by NightfallAlicorn** [comment]
Wouldn't it be better to know the source of the error, chance YouTube updated it's API and changed JSON output? I've just about got the update ready, about to have something to eat.

**Comment by Fuzzysqurl** [comment]
on a scale of 1 to 10, how many years would i have to wait to get this squished? :)

**Comment by NightfallAlicorn** [comment]
https://github.com/po-devs/po-server-goodies/pull/1311
closed and squished into single commit

**Pull request closed without merging**